src/assets/stylesheets/custom.scss

```diff
@@ -138,8 +136,39 @@ $textCenter: center;
 .db{ display: block; }
 .textCenter{ text-align: $textCenter; }
 .pointer{ cursor: pointer; }
+//patternfly-site.css
+h2{
+  font-size: em(18);
+  font-weight: 500;
+  line-height: 1.4;
+}
+//card-view
 
-//user -gravatar
+.card-pf-view {
+  .card-pf-top-element {
+    .card-pf-icon-circle {
+      @include Border(em(0.8), solid, $color-pf-blue-300); //#39a5dc;
+      @include borderRadius(50%);
+      @extend .db;
+      font-size: em(36);
+      height: em(34); //2.84em
```

Comment:
Marginally misses the UXD spec. it's 100x100px including border.
UXD specs it to be 102x102px (unsure with or without border).

I suggest apply em(36) to all, that'll *likely* match the spec.